torch/testing/_internal/common_methods_invocations.py

```diff
@@ -273,6 +279,8 @@ def method_tests():
         ('atan2', (S, 1, S), ((S, S),), 'broadcast_all'),
         ('reciprocal', torch.rand(S, S, S) + 0.1, NO_ARGS, '', (True,)),
         ('reciprocal', uniform_scalar(0.1, requires_grad=True), NO_ARGS, 'scalar', (True,)),
+        ('reciprocal', torch.randn(S, S, S, dtype=torch.cdouble) + 0.1, NO_ARGS, 'complex', (True,)),
+        ('reciprocal', uniform_scalar(0.1j, requires_grad=True), NO_ARGS, 'complex_scalar', (True,)),
```

Comment:
I'm not sure why the original test set requires_grad=True here.
You don't do it for any other tests so I think you can remove it.